### ISSUE TITLE
Bosh SHC slightly misleading description

### ISSUE THREAD

**Issue by Comodore125** [tschamm/boschshc-hass#197]
Hi,
I got Bosh SHC with 5 devices  + bridge. Had to move bridge closer and I got 100% reception. 
But I dont have the option to detect vibration inside sensors - I got those that support this. I see it only in bosh app. 
Also the alarm inside the app is not at all working and detected in Home Assistant. 

It seems that python integration should support these. https://github.com/tschamm/boschshcpy
Do you plan to add these two things? 

From my point of view, the intrusion alarm is not well made in the Bosh APP as it does not allow for audit trail or at least system log forwarding. Through home assistant my hope is to catch all this info into home assistant audit trail. 

If you dont have capacity to add this any chance to meet with you online in december? I should have some time and maybe I could add the code for that.

**Comment by mosandlt** [comment]
Vibration detection is supported, but only on the **Door/Window Contact II Plus** (`SWD2_PLUS`) — that's the model that exposes Bosch's `VibrationSensor` service, and the integration already creates a vibration `binary_sensor` for it. The regular **Door/Window Contact II** (`SWD2`) doesn't expose a vibration service at all (only ShutterContact + Bypass), so there's nothing for us to read on that model.

Could you confirm your exact device model (device info, or `bosch_shc.trigger_rawscan`)? If it's an `SWD2_PLUS` and you don't see the vibration sensor, that's a bug we can chase. For intrusion automations, the contact's open/closed state is already usable as a trigger today.

**Comment by Comodore125** [comment]
Yes, it is the contact II plus ( Bosch Smart Home Door/Window Contact II Plus, Burglary Protection with Smart Sensor for Vibration Detection, Compatible with Amazon Alexa and Google Assistant, White ) I have it for years. Works fine. Had to move the zigbee controller much closer but its performance is acceptable. But vibrations are not and never were propagated to the non-hacs version

**Comment by mosandlt** [comment]
Thanks for confirming it's the Contact II **Plus** (`SWD2_PLUS`). Key point: the vibration `binary_sensor` for `SWD2_PLUS` exists in **this HACS integration** (0.5.x) — but you mention the **non-HACS** (Home Assistant Core) version, which is an older snapshot that doesn't have it yet. If you install the HACS version (this repo) and restart, you should get a vibration binary_sensor for the device (it may start disabled in the entity registry — check "+N entities not shown"). Could you try that and confirm? If you're already on HACS 0.5.x and it's still missing, that's a bug and I'll dig into the vibration wiring.

**Comment by mosandlt** [comment]
I checked the wiring end-to-end: in **this HACS integration** the `SWD2_PLUS` vibration `binary_sensor` is created, enabled by default, and updates live (the entity subscribes to the `VibrationSensor` service, and `is_on` flips on `VIBRATION_DETECTED`). So it should work on the HACS build — the **non-HACS / Core** version is an older snapshot that doesn't include it.

The latest HACS release is now **0.5.2**. After updating, please also confirm **"vibration detection" is enabled for that contact in the Bosch Smart Home app** (it's a per-device toggle; if it's off, the SHC never sends vibration state). If you're on HACS 0.5.2 with detection enabled and the sensor still never goes `on`, reopen and I'll dig into it with a rawscan of the `VibrationSensor` service.

**Comment by Comodore125** [comment]
Hi, I am using haas integration for years and its still not there. I will try that in the near future. Had some different priorities. 
I did honestly not map things correctly and was not yet that sure what are the differences between versions in the past

**Comment by mosandlt** [comment]
No worries — and that version confusion is the likely culprit: there are **two** Bosch SHC integrations with the same name. The vibration sensor only exists in **this HACS one** (`tschamm/boschshc-hass`), *not* the built-in Home Assistant Core `bosch_shc`. So worth double-checking which one you actually have installed (HACS → Integrations → Bosch SHC should show version **0.5.2**).

When you get to it: update to 0.5.2, make sure **"vibration detection" is ON** for that contact in the Bosch app, then look for a **"Vibration"** binary_sensor on the device (it may be hidden under "+N entities not shown"). If it's still missing with all that, reopen and I'll dig in with a rawscan. No rush 🙂

**Comment by mosandlt** [comment]
Vibration detection (SWD2_PLUS) is supported in the HACS integration as a `binary_sensor` (≥0.5.0). Note this issue may have been filed against the HA Core integration, which lags behind HACS.

Available in current releases — please update to **boschshc-hass 0.7.6**. Closing; reopen if anything is missing.
